### ISSUE TITLE
XEM wrong returning "no show with the tvdb_id"

### ISSUE THREAD

**Issue by fernandog** [SiCKRAGETV/sickrage-issues#2300]
{"result":"failure","data":[],"message":"no show with the tvdb_id 274431 found"}

2015-07-23 03:46:20 DEBUG    SHOWQUEUE-FORCE-UPDATE :: Requested getURL http://thexem.de/map/all?id=274431&origin=tvdb&destination=scene returned status code is 200

BUT
http://thetvdb.com/?tab=series&id=274431

We are doing like the docs
all other shows result in the same message.

**XEM DOC**:
http://thexem.de/doc
All
Basically same as "single" just a little easier
The origin address is added into the output too!!
http://thexem.de/map/all?id=7529&origin=anidb

**Comment by fernandog** [comment]
asked in #xem IRC. waiting for a reply

**Comment by fernandog** [comment]
they suggested to see if show has mapping before query the URL with indexer_id:

http://thexem.de/map/havemap?origin=tvdb 

{"result":"success","data":["2797","2801","30203","31395","31397","31398","31399","70668","70900","71361","71470","71634","72241","72454","72902","73141","73388","73532","73749","73871","74379","74618","74709","74718","74796","74852","75205","75411","75863","75869","75886","76023","76085","76156","76168","76666","76703","76705","76884","76894","76932","77068","77681","77992","78282","78500","78812","78857","78886","78904","78914","78916","78919","78920","78962","78972","78998","79017","79099","79106","79151","79156","79178","79203","79206","79214","79283","79360","79460","79525","79604","79637","79682","79685","79719","79824","79839","79857","79886","79895","80042","80044","80206","80327","80328","80552","80554","80644","80646","80660","80747","80808","80830","80900","80904","81108","81110","81115","81178","81184","81234","81463","81766","81782","81797","81807","81831","82135","82154","82207","82266","82548","82551","82788","82807","83105","83231","83314","83322","83360","83363","83475","83996","84218","84230","84591","84709","87491","87501","87841","87881","88031","88391","101501","102041","102261","105611","111051","114801","114921","116001","116291","117801","118101","123611","129261","132961","133341","134181","139481","140071","141181","145211","146401","146711","148561","164091","166251","171591","171731","172921","173211","173271","173301","177151","186911","188551","190351","193711","193811","194961","195571","195711","195721","199011","199051","207621","208111","212551","213221","219131","219701","219771","220571","224011","227891","240291","244161","245821","246521","247836","247911","247921","247934","247958","248718","248962","249827","249865","249867","249869","249872","250022","250071","250181","251085","251906","251908","251982","252133","252322","252343","252345","252423","252474","254650","254652","254653","254658","254662","254663","254667","254669","254738","254798","254931","255056","255206","256394","256705","257098","257517","257521","257522","257571","257645","257756","257762","257765","257837","257875","257881","258480","259393","259635","259636","259638","259640","259641","259642","259643","259645","259649","259652","259653","259675","260634","260946","260995","261511","261862","262090","262092","262093","262094","262106","262107","262109","262112","262113","262114","262838","262856","262954","263688","264047","264049","264050","264051","264052","264055","264056","264522","264523","264527","264528","264531","264532","264533","264663","264972","265530","265720","266449","267246","267432","267433","267435","267436","267439","267440","267441","267543","267649","267705","267810","268059","268060","268284","268286","268289","269084","269285","269319","269681","269682","269872","269873","269876","270057","270058","270062","270064","270065","270068","270202","270701","271152","271591","271659","271687","271708","271813","271984","272073","272128","272138","272156","272309","272312","272314","272316","272317","272472","272697","272849","273002","273004","273005","273009","273616","274886","275039","275573","275574","275579","275583","275613","275614","275625","275628","275629","275633","275634","275670","276163","276345","276896","277464","278126","278127","278151","278152","278153","278154","278156","278196","278199","278262","278327","278329","278330","278331","278626","278817","278818","278819","278941","278975","278976","279040","279042","279396","279527","279528","279530","279533","279804","279826","279827","279830","279831","279979","279980","279981","279984","279986","280035","281249","281275","281947","281949","282060","282899","282994","283073","283115","283277","283468","283611","283924","283925","283928","283930","283931","283937","283942","283943","283946","283947","283963","284131","284132","284133","284169","284211","284263","284402","284439","284563","284585","284601","284641","284643","284719","284794","284943","285357","286855","286856","286987","288103","288914","289172","289174","289177","289310","289585","289679","289744","289763","289882","289884","289885","289886","289903","289905","289906","289907","289909","290245","290561","291625","291629","291630","292098","292567","292689","293119","293121","293267","293647","293774","294001","295204","295205","295220","295222","295243","295896","296713","296764","297315","414921","514046"],"message":"These shows have some kind of mapping"}

**Comment by fernandog** [comment]
start point:
https://github.com/SiCKRAGETV/SickRage/blob/develop/sickbeard/scene_numbering.py
